backend/typescript/migrations/20220420005809-change_book.seriesOrder_from_string_to_int.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+"use strict";
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return Promise.all([
+      queryInterface.changeColumn("books", "series_order", {
+        type: 'INTEGER USING CAST("series_order" as INTEGER)',
+        // allowNull: true
+      }),
+    ]);
+  },
+
+  async down(queryInterface, Sequelize) {
+    return Promise.all([
+      queryInterface.changeColumn("books", "series_order", {
+        type: 'STRING USING CAST("series_order" as STRING)',
```

Comment:
Migrating down doesn't work. I don't think Postgres has a "STRING" type. Could you check what it was before (either `CHAR` or `VARCHAR`)